daemon/src/db.rs

```diff
@@ -160,6 +173,14 @@ impl Connection {
         .bind(&cfd.settlement_time_interval_hours().whole_hours())
         .bind(&cfd.quantity())
         .bind(&cfd.counterparty_network_identity())
+        .bind(&cfd.counterparty_libp2p_identity().unwrap_or({
+            tracing::debug!(
+                order_id=%cfd.id(),
+                taker_id=%cfd.counterparty_network_identity(),
+                "Inserting deprecated CFD with placeholder peer-id!"
```

Comment:
🙃 The exclamation mark makes me think this is very interesting, but I think this is pretty boring.

```suggestion
                "Inserting deprecated CFD with placeholder peer-id"
```